recipe/meta.yaml

```diff
@@ -33,18 +43,16 @@ requirements:
 
 test:
   requires:
-    - cryptography-vectors {{ version }}
-    - hypothesis >=1.11.4,!=3.79.2
-    - iso8601
+    - cryptography-vectors == {{ version }}
     - pip
-    - pretend
     - pytest >=6.2.0
-    - pytest-subtests
-    - pytest-xdist
     - pytest-benchmark
-    - pytz
+    - pytest-cov
```

Comment:
Is `pytest-cov` really needed?

Reply:
It is by spec here: https://github.com/pyca/cryptography/blob/d02de9f26e9a2353e89427c1cea8b9ed2bae969e/pyproject.toml#L73

Though you are right we don't look at the results.